.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "glint.libraryPath": "ember-bootstrap"
```

Comment:
The Glint VSCode plugin expect to find the glint libraries in the root's node_modules, where they aren't due to pnpm not hoisting dependencies. So this is making the plugin work ootb when setting up the repo locally. Normally, I wouldn't but code editor configs into the repo, but in this case maybe it makes sense?